tests/app/v2/notifications/test_get_notifications.py

```diff
@@ -339,6 +339,51 @@ def test_get_all_notifications_filter_by_single_status(api_client_request, sampl
     assert json_response["notifications"][0]["status"] == "pending"
 
 
+@pytest.mark.parametrize(
+    "internal_status, filter_status, should_get",
```

Comment:
This `should_get` variable has a pretty confusing name, and I struggled to find a better one, and I thought - does this mean maybe we could approach this parametric in another way?

For example if you changed this param to `expected_num_of_notifications` instead, it would be either 0 or 1, and you could use it for your first assertion, too, and it would be easier to read.

And then the test would be also checking that there are no false positives ☺️ .